packages/relative-values/src/components/View/ListView/sidebar.tsx

```diff
@@ -0,0 +1,113 @@
+import { deflate } from "pako";
+import { ModelEvaluator } from "@/values/ModelEvaluator";
+import { Item } from "@/types";
+import { fromByteArray } from "base64-js";
+import { getModelCode } from "@/model/utils";
+import { FC, Fragment } from "react";
+import { NumberShower } from "@quri/squiggle-components";
+import { SquiggleChart } from "@quri/squiggle-components";
+
+function setHashData(str: string) {
+  const text = JSON.stringify({ initialSquiggleString: str });
+  const HASH_PREFIX = "https://www.squiggle-language.com/playground#code=";
+  const compressed = deflate(text, { level: 9 });
+  return HASH_PREFIX + encodeURIComponent(fromByteArray(compressed));
```

Comment:
Playground URLs are kind of a public API at this point, maybe we should move this function to `squiggle-lang`.

Reply:
I was thinking about that. Not sure what's best. It doesn't seem like a huge deal if we don't have to really support multiple environments too well.